### PR TITLE
Clarify usage of transport hints

### LINKED ISSUE #215: Figure out how transport hints get from Authenticator to RP
Is it via some metadata service?

Via attestation data?  <= my vote is on this

### PR DESCRIPTION
**Pull request by leshi**
Closes #215

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -872,16 +872,21 @@ these methods.
 </pre>
 
 <div dfn-for="Transport">
-    This enumeration defines the valid transports over which a client might
-    look for an Authenticator that can provide an assertion for the respective
-    credential.  Note that these are a hint from the Relying Party to the
-    client as to its best belief of how an authenticator may be reached.
+    Non-platform Authenticators communicate with Clients using a variety of
+    transports.  This enumaration defines a hint as to how Clients might
```
Comment (vijaybh) [inline index.bs]: Typo - enumeration

File: index.bs [hunk 2]
```diff
@@ -872,16 +872,21 @@ these methods.
 </pre>
 
 <div dfn-for="Transport">
-    This enumeration defines the valid transports over which a client might
-    look for an Authenticator that can provide an assertion for the respective
-    credential.  Note that these are a hint from the Relying Party to the
-    client as to its best belief of how an authenticator may be reached.
+    Non-platform Authenticators communicate with Clients using a variety of
```
Comment (vijaybh) [inline index.bs]: Have we defined "non-platform authenticator" up to this point? Maybe reword to be more easily understood by novice reader?

**Comment by vijaybh** [comment]
Also please merge from master and resolve conflicts.

File: index.bs (hunk 1 above)
Comment (leshi) [inline index.bs]: done

File: index.bs (hunk 2 above)
Comment (leshi) [inline index.bs]: See if you like this better

**Comment by leshi** [comment]
Thanks for the review!  Please take another look.

**Comment by leshi** [comment]
gah, something doesn't build, hold tight.

**Comment by leshi** [comment]
Okay, I have no idea why travic-cl threw up, but this seems to build locally, and I cannot see any issues with the commit.  This is ready for review.

**Comment by vijaybh** [comment]
Travis is failing to sync Bikeshed correctly. Let me poke at it later today. In the meantime I wonder (though I am not sure so I'll defer to your judgment) if we could also add a sentence here explaining the absence of an "internal" transport. Other than that this LGTM.

**Comment by vijaybh** [comment]
OK, master should now be fixed to avoid these errors. Would you please merge again and confirm that this PR builds?

**Comment by equalsJeffH** [comment]
looks fine to me.

**Comment by leshi** [comment]
Travis is happy again.  Thanks, Vijay!

Decided to not to expand on lack of "internal" because I could not describe it in once sentence without mentioning the ill defined "non-platform/platform authenticator".

**Comment by leshi** [comment]
Merging as I've gotten an LGTM from Jeff and Vijay
